inc/stm32flash.h

```diff
@@ -207,6 +208,46 @@
 
 #define STM32L4_FLASH_OPTR_DUALBANK 21
 
+// Flash registers common to STM32L5 series
+// RM0438, pg. 241
+#define STM32L5_FLASH_REGS_ADDR ((uint32_t)0x50022000)
```

Comment:
We should use the 0x40022000 base address. The 0x50022000 address used in the security mode (when the TrustZone is enabled). RM0438, pg. 2194